frontend/src/components/CreateMemo/components/LocationMapPicker/Map.tsx

```diff
@@ -3,26 +3,32 @@ import { useState, useEffect } from "react";
 import { MapContainer, TileLayer, Marker, Popup, useMapEvents } from "react-leaflet";
 import L from "leaflet";
 import markerIcon from "/images/marker-icon.png";
-const MapForm = ({ selectedLocation, onMapClick }: any) => {
-  const [initialPosition, setInitialPosition] = useState([49.27326489299744, -123.10365200042726]);
-  const [position, setPosition] = useState(initialPosition);
+
+const MapForm = (props:{ 
+    selectedLocation:string, 
+    onMapClick:([number,number])=>void
+  }) => {
+
+  const [initialPosition, setInitialPosition] = useState<[number,number]>([49.27326489299744, -123.10365200042726]);
+  const [position, setPosition] = useState<[number,number]>(initialPosition);
 
```

Comment:
extract out the default coordinates into a constant, `initialPosition` is sorta redundant here, use that constant instead

Reply:
updated